client/my-sites/hosting/sftp-card/index.js

```diff
@@ -187,10 +187,19 @@ export const SftpCard = ( {
 					checked={ isSshAccessEnabled }
 					onChange={ () => toggleSshAccess() }
 					label={ translate(
-						'Enable SSH access for this site. {{em}}This feature is currently in beta.{{/em}}',
+						'Enable SSH access for this site. {{em}}This feature is currently in beta.{{/em}} For more information see {{supportLink}}SSH on WordPress.com{{/supportLink}}.',
 						{
 							components: {
 								em: <em />,
+								supportLink: (
+									<ExternalLink
+										icon
+										target="_blank"
```

Comment:
Personally I'd remove these `target` props since they aren't needed. `ExternalLink` implies `target=_blank`

Reply:
> Personally I'd remove these `target` props since they aren't needed. `ExternalLink` implies `target=_blank`

Interestingly, while `ExternalLink` _implies_ `target=_blank`, it doesn't actually default to `target=_blank` 😦 

Instead, `target=_blank` is conditionally applied based on `icon`:

https://github.com/Automattic/wp-calypso/blob/6304534b72bdc698bee159c30459c00f198a9aac/client/components/external-link/index.jsx#L40-L42

While we are using an `icon` in this instance, I'd prefer to keep the `target=_blank` as an explicit reference.